ims_app/src/components/screens/inventories/inventories.js

```diff
@@ -3,7 +3,7 @@ import React, { Component } from 'react';
 class Inventories extends Component {
     render() {
         return (
-            <div className='page'>
+            <div className='categories'>
```

Comment:
why did you rename this to categories?

Reply:
It must have been a mistake. I have effected the correction